### PR TITLE
Fixing associations on com_menus side-by-side

### PR DESCRIPTION
**Pull request by jreys**
Pull Request for Issue # .
#### Summary of Changes

Associations now correctly saved and checkin is confirmed by applying PR #46

Notice:
- Editing associations on the reference and changing the language won't apply the change but since those fields will be readonly there's no problem.

### CODE REVIEW

File: administrator/components/com_associations/views/association/tmpl/edit.php [hunk 1]
```diff
@@ -219,14 +232,13 @@
...
 $aView      = $input->get('aview', '', 'string');
-$extension  = $input->get('extension', '', 'string');
 $rLanguage  = $input->get('referencelanguage', '', 'string') != null ? $input->get('referencelanguage', '', 'string') : '';
 ?>
 <button id="toogle-left-panel" class="btn btn-small">Show/Hide Reference (PoC)</button>
 
 <form action="<?php echo JRoute::_(
 			'index.php?option=com_associations&view=association&layout=' . $layout . '&acomponent='
-			. $aComponent . '&aview=' . $aView . '&extension=' . $extension . '&referencelanguage=' . $rLanguage . '&id='
+			. $aComponent . '&aview=' . $aView . '&referencelanguage=' . $rLanguage . '&id='
```
Comment (andrepereiradasilva) [inline administrator/components/com_associations/views/association/tmpl/edit.php]: Do not remove this

File: administrator/components/com_associations/views/association/tmpl/edit.php [hunk 2]
```diff
@@ -219,14 +232,13 @@
 $layout     = $input->get('layout', '', 'string');
 $aComponent = $input->get('acomponent', '', 'string');
 $aView      = $input->get('aview', '', 'string');
-$extension  = $input->get('extension', '', 'string');
```
Comment (andrepereiradasilva) [inline administrator/components/com_associations/views/association/tmpl/edit.php:222]: Do not remove this

File: administrator/components/com_associations/views/association/tmpl/edit.php [hunk 3]
```diff
@@ -121,27 +121,37 @@
...
+						// - For modal association selectors.
 	 					langAssociation = parse[0].replace(/-/,'_');
 	 					target.find('#jform_associations_' + langAssociation + '_id').val(parse[1]);
+
+	 					// - For chosen association selectors (menus).
+						target.find('#jform_associations_' + langAssociation + '_chzn').remove();
+						chznField = target.find('#jform_associations_' + langAssociation);
+						chznField.append('<option value=\"'+ parse[1] + '\"></option>');
+						chznField.val(parse[1]).change().chosen();
 	 				}
```
Comment (andrepereiradasilva) [inline administrator/components/com_associations/views/association/tmpl/edit.php:153]: Doing si!ikar operatons before and in the cycle. Can't we do all inside the the cycle

Comment (jreys) [inline administrator/components/com_associations/views/association/tmpl/edit.php:153]: Nope, the cycle goes trough every language except the reference language, that's why it's separated

Comment (andrepereiradasilva) [inline administrator/components/com_associations/views/association/tmpl/edit.php:153]: Makes sense,!

**Pull request merged**